internal/config/config.go

```diff
@@ -52,8 +52,25 @@ func (c *Config) Validate() error {
 type Flags struct {
 	// Version decides whether to just print the version and exit.
 	Version bool `long:"version" description:"print version and exit"`
-	// Config is the path to the config file
-	Config string `short:"c" long:"config" description:"path to config file"`
+
+	// Config is the path to the config file. If not provided, it defaults to DefaultConfigPath.
+	Config string `short:"c" long:"config" description:"path to config file" default:"/etc/icingadb/config.yml"`
+	// default must be kept in sync with DefaultConfigPath.
+}
+
+// GetConfigPath retrieves the path to the configuration file.
+// It returns the path specified via the command line, or DefaultConfigPath if none is provided.
+func (f Flags) GetConfigPath() string {
+	if f.Config == "" {
+		return DefaultConfigPath
+	}
+
+	return f.Config
+}
+
+// IsExplicitConfigPath indicates whether the configuration file path was explicitly set.
+func (f Flags) IsExplicitConfigPath() bool {
+	return f.Config != ""
```

Comment:
As commented above, this does not seem to work.

Thus, I am unable to parse a configuration based solely on environment variables as the code always bails out due the unavailability of the `/etc/icingadb/config.yml` file, <https://github.com/Icinga/icinga-go-library/blob/7ce60227c33d68d1a9d5a958a2c2f9409b05168c/config/config.go#L273>.